### PR TITLE
Gruvbox light theme version

### LINKED ISSUE #3: Gruvbox light theme version
### Description
As a developer,

I want to use a light theme Gruvbox for Tmux

so that it can better fit my light terminal theme.

### Acceptance

- [x] implement light Gruvbox theme
- [x] adjust color to nicely fit the light Gruvbox theme for vim and terminal itself

### Additional Information

*   Depends on issue(s): -
*   Related to issue(s): -

### PR DESCRIPTION
**Pull request by egel**
closes #3

- add basic .gitignore
- gruvbox light theme for tmux

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,17 +19,21 @@ cat tmux-gruvbox-dark.conf >> ~/.tmux.conf
...
+
+Add plugin to the list of TPM plugins in `.tmux.conf` and select desired theme
+
+```bash
 set -g @plugin 'egel/tmux-gruvbox'
+set -g @tmux-gruvbox 'dark'
 ```
-Hit `prefix + I` to fetch the plugin and source it.
-Your tmux should be updated with the theme at this point.
+
```
Comment (egel) [inline README.md:29]: Still missing of docs for optional subthemes

**Comment by saragiotto** [comment]
Hi @egel, how are you? This is a very helpful PR. Could please merge it? 🙏

**Comment by egel** [comment]
Hi @saragiotto. I'm fine, thanks 👍  

I will take a look at it as soon as I can and if something will not missing I will release it 🙂

File: README.md (hunk 1 above)
Comment (egel) [reply README.md:31]: done

**Pull request merged**